### DISCUSSION TITLE
forking is taking too long to response.

### DISCUSSION THREAD

**Discussion by nonjee888** [community/community#47132] [Repositories] [answered]
I tried to fork but it took too long to load the page, any solution can be shared?

**Comment by KevinLuna95** [comment]
I have the same error. The message are: 
This page is taking too long to load.

Sorry about that. Please try refreshing and contact us if the problem persists.

i try to fork the next repository: https://github.com/ibm-developer-skills-network/jbbmo-Introduction-to-Git-and-GitHub

**Reply by nonjee888** [reply]
exactly the same page was loaded to me too.

**Reply by viraj-s15** [reply]
I have the exact same problem. Anyone found a solution yet?

**Reply by weidehai** [reply]
me too

**Reply by soaringpanda** [reply]
same issue here!

**Reply by metinc** [reply]
Same here. I also tried using the Github API. This is the response I get...
`{
  "message": "Server Error"
}`

**Reply by TaeyanG4** [reply]
me too... it's maybe git's problem

**Reply by Jacobtims** [reply]
I have the exact same problem with https://github.com/laravel/laravel

**Reply by danilowhk** [reply]
Having the same issue here

**Reply by OpenSauce04** [reply]
Having the same issue as well, something must be wrong on GitHub's end, too many people are having this happen at the same time
https://github.com/JuliaLang/www.julialang.org for me

**Reply by yahao87** [reply]
me too...

**Reply by ahhmedsafwat** [reply]
I think its a common error cuz it's giving me the same error every time I try to create a fork

**Reply by tlm159** [reply]
Pareil ici, j'essaye depuis ce matin.

**Reply by ketan-chirde** [reply]
I have the same exact problem with https://github.com/kurtmkurtm/LiquidTestReports

**Comment by danilowhk** [comment]
This is the repo I am trying to fork: https://github.com/keep-starknet-strange/quaireaux/

**Comment by xbryanxx** [comment]
same issue

**Comment by YuriNek0** [comment]
Same issue here.

**Comment by ahhmedsafwat** [comment]
it gives me this error every time I try to get a repo, "This page is taking too long to load. "

**Comment by Jacobtims** [comment]
It gave me a 500 SERVER ERROR a moment ago.

**Comment by grentank** [comment]
I have the very same problem. Can't fork repositories or even browse them

**Comment by cary-rowen** [comment]
I have the very same problem. Can't fork repositories.

**Comment by IuliiaSukhinina** [comment]
I have the same error. Can't fork repositories.

**Comment by rinarakaki** [comment]
Same.

**Comment by Jacobtims** [comment]
Fixed for me

**Reply by metinc** [reply]
Can confirm.

**Reply by OpenSauce04** [reply]
Same

**Comment by tlm159** [comment]
c'est bon pour moi :+1:

**Comment by queenofcorgis** [comment]
Hi there! Can folks confirm if this is still persisting for them - I've let our team know about these posts! 

@nonjee888,  you are more likely to get a useful response if you are posting your question in the applicable category, the Issues category is solely related to conversations around the GitHub products Issues and Projects. I've gone ahead and moved it for you. Good luck!

**Reply by nonjee888** [reply]
Thanks for your support :)

**Comment by queenofcorgis** [comment, marked as answer]
This is now resolved - as we are not seeing these errors show up in our monitoring. Our monitors show this event happened February 12  between 6-8am ET. However, if you are still seeing this persist or have had this occur more recently, please let me know ❤️ .

**Comment by amirhosseinep** [comment]
I have the same issue forking from a repository!

**Comment by amirhosseinep** [comment]
I have the same issue forking from a repository!

**Comment by Jacobtims** [comment]
https://www.githubstatus.com/

**Comment by dodrin** [comment]
I have been having the same issue forking a gist...
